rpc/manager/manager.proto

```diff
@@ -88,8 +88,8 @@ message InterceptSpec {
   string target_host = 6;
   int32 target_port = 7;
 
-  // Name of ther service port that is being intercepted
-  string service_port_name = 10;
+  // Identifier for the service port: either the name or port number
+  string service_port_identifier = 10;
```

Comment:
I thought about making this servicePortNameOrNumber to match how it's referenced later in the connector code, but thought that might be a really long rpc name, but if we don't think that's a concern, I can change this so it's the same throughout.